gog/graph.go

```diff
@@ -47,6 +47,16 @@ type Grapher struct {
 
 	seenDocObjs map[types.Object]struct{}
 	seenDocKeys map[string]struct{}
+
+	// pkgDefs accumulates the defs from the package being graphed currently in Graph().
+	// If the package is graphed successfully, the defs are added to Output.
+	pkgDefs []*Def
```

Comment:
Sorry, I misspoke. `func (g *Grapher) Graph` inside `gog` graphs an individual package. Why not have these as local vars in that method instead of fields on the struct?